doc/jbuild.rst

```diff
@@ -1088,6 +1122,31 @@ is global to all build contexts, simply use an absolute filename:
 
 .. _ocaml-syntax:
 
+Promotion
+---------
+
+The ``(promote (<file1> as <file2>) (<file3> as <file4>) ...)`` action
+can be used to copy generated files to the source tree.
+
```

Comment:
Shouldn't `promote-if` also be mentioned here?

Reply:
Indeed